lambdas/gro-pull-file/src/main/java/uk/gov/di/data/lep/GroPullFile.java

```diff
@@ -79,27 +87,38 @@ private void transferFile(String xmlBucket) {
             client.connect(host);
             client.authPublickey(username, privateKeyProvider);
 
-            logger.info("Pulling file {} from GRO", groFileName);
             try (var sftpClient = client.newSFTPClient()) {
                 var resources = sftpClient.ls(sourceDir);
-                logger.info(resources);
```

Comment:
You can leave this one deleted though - this got put in just before all of the hmpo mess and then I forgot to take it out again 😅